### ISSUE TITLE
Loss is not reducing

### ISSUE THREAD

**Issue by Qtsho** [Qtsho/model-based-rl-navigation#4]
The loss is reducing at the first phase of training, I guess due to overfitting. However, when we have larger iteration, the loss has a very high variance.

**Comment by Qtsho** [comment]
loss shoot up after running a long time to collect data.
![image](https://user-images.githubusercontent.com/37075262/140488336-7abaf9cb-f294-4a96-ab8a-260d52d98feb.png)
